javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,15 @@
+var $chars = document.querySelectorAll('span');
+
+var charsArr = Array.from($chars);
+
+var currentIndex = 0;
+
+document.addEventListener('keydown', function (event) {
+  charsArr[currentIndex + 1].className = 'underscore';
+  if (charsArr[currentIndex].textContent === event.key) {
+    charsArr[currentIndex].className = 'correct';
+    currentIndex++;
+  } else {
+    charsArr[currentIndex].className = 'wrong';
```

Comment:
If a character is incorrect, the user should have to retype that character until it is correct, the typing app shouldn't leave the letter wrong and move on.